### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

**Comment by rex4539** [comment]
@sisou Could you please take a look?

File: scripts/devnet/node.py [hunk 1]
```diff
@@ -319,7 +319,7 @@ def poll(self):
 
         :return: The return code of the process if it ended, None if it hasn't
             finished
-        :rtype: int ot None
+        :rtype: int to None
```
Comment (jsdanielh) [inline scripts/devnet/node.py]:
```suggestion
        :rtype: int or None
```

**Review by jsdanielh — CHANGES_REQUESTED** [review]
Thanks for the PR. I only noted a wrong fix. Could you fix it?

File: scripts/devnet/node.py (hunk 1 above)
Comment (rex4539) [reply scripts/devnet/node.py]: Fixed and rebased.

**Review by jsdanielh — APPROVED** [review]

**Comment by jsdanielh** [comment]
@rex4539 Will push an additional commit since you just uncovered a compilation error that was hidden under the `fature` typo.

**Pull request merged**